src/vivarium_public_health/metrics/stratification.py

```diff
@@ -0,0 +1,353 @@
+from dataclasses import dataclass
```

Comment:
I think this will break 3.6 support without a future import, I'm pretty sure.

Reply:
You are correct. There is a backport for `dataclasses` which appears to be legit. Users shouldn't be using 3.6 ideally, but this worked with the backport when I tested it.